testctrl/proto/test_sessions.proto

```diff
@@ -0,0 +1,185 @@
+// Copyright 2020 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package proto;
+
+option go_package = "github.com/grpc/grpc/testctrl/genproto/testctrl/svc";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "google/longrunning/operations.proto";
+import "src/proto/grpc/testing/control.proto";
+
+service TestSessions {
```

Comment:
I started with the `Service` suffix, and I agree it provides a level of clarity. That said, I removed it because the Google API's long-running service is called `Operations` without the suffix. For consistency, I thought I'd keep both the same. What are your thoughts, @jtattermusch / @paulosjca ?

Reply:
I'd stick with whatever services under https://github.com/grpc/grpc/tree/master/src/proto/grpc do (I think they all use the suffix).

Reply:
Judging from the practice in gRPC team, I would add the Service suffix. It seems that the practice is to name the package without "service" and only add it to the actual service definition, so if we have BenchmarkSessionManagerService I would expect that we would ultimately have the proto package be grpc.testing.benchmarking.benchmarksessionmanager.v1 and perhaps even the go package become something similar, with "testctrl" replaced by "benchmarksessionmanager"  (or perhaps just "sessionmanager"?). For now, since we are committing in a private fork, we can go either way, and discuss it later.